WordPress/src/main/java/org/wordpress/android/ui/plans/util/IabHelper.java

```diff
@@ -1025,11 +1038,11 @@ public void run() {
     }
 
     void logDebug(String msg) {
-        AppLog.d(AppLog.T.PLANS, msg);
+        if (mDebugLog) AppLog.d(AppLog.T.PLANS, msg);
     }
 
     void logError(String msg) {
-        AppLog.d(AppLog.T.PLANS, "In-app billing error: " + msg);
+        AppLog.e(AppLog.T.PLANS, "In-app billing error: " + msg);
```

Comment:
The original version doesn't have the  `mDebugLog` check for `w` and `e` logging. I think it's ok not having it there, since w and e are important.